### PR TITLE
Loading component

### PR DESCRIPTION
**Pull request by Dalmano**
- Relates #61

### CODE REVIEW

File: client/src/components/Loading/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
```
Comment (AbdallahAmmar96) [inline client/src/components/Loading/index.js]: add new line

File: client/src/components/Loading/index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+import "./style.css";
+
+export default ({ className }) => {
```
Comment (AbdallahAmmar96) [inline client/src/components/Loading/index.js]: Why is the className here ? 
I think it will be standard for all pages, am I right?

Comment (dubhcait) [reply client/src/components/Loading/index.js]: We'd only need two if we want one for the popup. It would need to be smaller I think. What do we think?

Comment (AbdallahAmmar96) [reply client/src/components/Loading/index.js]: ok, that's okay
just add a new line after importing the react

**Pull request merged**